cocos/render-scene/scene/model.ts

```diff
@@ -678,8 +676,7 @@ export class Model {
         if (!this._localDataUpdated) { return; }
         this._localDataUpdated = false;
 
-        // @ts-expect-error using private members here for efficiency
-        const worldMatrix = this.transform._mat;
+        const worldMatrix = this.transform.worldMatrix;
```

Comment:
_mat is marked as `engineInternal` and `public` now, should we revert this line?

Reply:
reverted
https://github.com/cocos/cocos-engine/pull/14572/commits/4f5e07fb7df74d1c38b205fcd5af5b160234fa82